### PR TITLE
Qubit Segments

### PR DESCRIPTION
**Pull request by jhollingworth**
We store your current segments in stash which looks a bit like this:

```
{
  status: {
   code: 200
  },
  data: "{"segments":{"SG-3918-f06a1268":true},"version":1}"
}
```

This is a simple PHP library that wraps getting memberships from stash + provides a few helper methods (e.g. getting your visitor Id from the cookie)

### CODE REVIEW

**Comment by jhollingworth** [comment]
@afternoon @rogy @KidkArolis can I get a review please

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,9 @@
+build/*          export-ignore
+docs/*           export-ignore
+tests/*          export-ignore
+.editorconfig    export-ignore
+.gitattributes   export-ignore
+.gitignore       export-ignore
+.travis          export-ignore
+Makefile         export-ignore
+phpunit.xml.dist export-ignore
```
Comment (Olical) [inline .gitattributes]: What does all this mean?

File: build/packager.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+// Copy each dependency to the staging directory. Copy *.php and *.pem files.
+$packager->recursiveCopy('src', 'QubitSegments', ['php']);
+
+$packager->createAutoloader([
+    'QubitSegments/functions.php',
+]);
+
+$packager->createPhar(__DIR__ . '/artifacts/qubit-segments.phar');
+$packager->createZip(__DIR__ . '/artifacts/qubit-segments.zip');
```
Comment (Olical) [inline build/packager.php]: Is this and the burger thing just for packaging? Why is it part of the repo? :S

Comment (jhollingworth) [inline build/packager.php]: I did a copy + paste of guzzle. its used for publishing to composer (I think)

Comment (jhollingworth) [inline build/packager.php]: removed in 6d45971

File: .gitattributes (hunk 1 above)
Comment (jhollingworth) [inline .gitattributes]: No idea, removed 45522e5

**Comment by jhollingworth** [comment]
I need to merge but please review and I will update in a second PR

**Comment by rogy** [comment]
It generally looks fine to me, the actual segments stuff is fairly straightforward anyway.

**Pull request merged**